### ISSUE TITLE
Region Plugins: Cannot read property 'disable-drag-selection' of undefined

### ISSUE THREAD

**Issue by vuraldincer** [katspaugh/wavesurfer.js#2052]
Hi,

**I get an error that I don't get while using 3.3.3 version in 4.0.1 version and**
```js
...
wavesurfer = WaveSurfer.create({
 ...
 plugins: [
                    WaveSurfer.cursor.create({
                        showTime: false,
                        color: 'hsla(145, 63%, 49%, 1)',
                        customStyle: {'margin-left': '80px'},
                        customShowTimeStyle: {'margin-left': '-25px', 'background-color': 'hsla(200, 5%, 66%, .8)'}
                    }),
                    WaveSurfer.regions.create({
                        dragSelection: {
                            slop: 3
                        }
                    })
      ]
});
**When using the "regions" library**
```js
wavesurfer.enableDragSelection ({}); //applying then
wavesurfer.addRegion ({...});
```
**when i do i get the following error**
_Uncaught TypeError: Cannot read property 'disable-drag-selection' of undefined
     at e.value [as fireEvent] (observer.js: 131)
     at e.value (index.js: 377)
     at e.value (index.js: 216)
     at n._onReady (index.js: 145)
     at r (observer.js: 90)
     at observer.js: 132
     at Array.forEach (<anonymous>)
     at n.value (observer.js: 132)
     at s. <anonymous> (wavesurfer.js: 1456)
     at r (observer.js: 90)_

how should i go ?
_Version 84.0.4147.135 (Official Build) (64-bit) On Windows_

_Originally posted by @vuraldincer in https://github.com/katspaugh/wavesurfer.js/issues/1861#issuecomment-681834379_

**Comment by Dr-Snuggles** [comment]
Noticed this error when updating to a newer version of wavesurfer in my app also.
The same problem occurs on the documentation page too.
See [https://wavesurfer-js.org/plugins/regions.html](https://wavesurfer-js.org/plugins/regions.html)

**Comment by Dr-Snuggles** [comment]
Should probably add that the problem occurs after upgrading from WaveSurfer 4.0.1 to 4.1.0.

**Comment by todoroff** [comment]
> Should probably add that the problem occurs after upgrading from WaveSurfer 4.0.1 to 4.1.0.

Same here

**Comment by Lightn** [comment]
Same as #2067, which was fixed in 4.1.1.
